src/style.css

```diff
@@ -0,0 +1,58 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 2px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--1 :not(:nth-child(n + 2)),
+.stars--2 :not(:nth-child(n + 3)),
+.stars--3 :not(:nth-child(n + 4)),
+.stars--4 :not(:nth-child(n + 5)),
+.stars--5 :not(:nth-child(n + 6)) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+/* 
+.stars--0 :nth-child(n + 2),
+.stars--1 :nth-child(n + 2),
+.stars--2 :nth-child(n + 2),
+.stars--3 :nth-child(n + 2),
+.stars--4 :nth-child(n + 2),
+.stars--5 :nth-child(n + 2) {
+  padding-left: 2px;
+}
+
+.stars--0 :not(:last-child),
+.stars--1 :not(:last-child),
+.stars--2 :not(:last-child),
+.stars--3 :not(:last-child),
+.stars--4 :not(:last-child),
+.stars--5 :not(:last-child) {
+  padding-right: 2px;
+} */
+
+.stars--0 :nth-child(n),
+.stars--1 :nth-child(n),
+.stars--2 :nth-child(n),
+.stars--3 :nth-child(n),
+.stars--4 :nth-child(n),
+.stars--5 :nth-child(n) {
+  padding-right: 2px;
+  padding-left: 2px;
```

Comment:
delete it, just add margin-right for stars__star

Reply:
No, code for 
.stars__star {
  background-image: url(images/**star.svg**);
  background-repeat: no-repeat;
}
at the same time, code for 
.stars--4 :not(:nth-child(n + 5)),
.stars--5 :not(:nth-child(n + 6)) {
  background-image: url(images/**star-active**.svg);
  background-repeat: no-repeat;
}

these selectors have different images  active or not-active